styles/hoogle/catppuccin.user.css

```diff
@@ -0,0 +1,327 @@
+    @rosewater: #dc8a78; @flamingo: #dd7878; @pink: #ea76cb; @mauve: #8839ef; @red: #d20f39; @maroon: #e64553; @peach: #fe640b; @yellow: #df8e1d; @green: #40a02b; @teal: #179299; @sky: #04a5e5; @sapphire: #209fb5; @blue: #1e66f5; @lavender: #7287fd; @text: #4c4f69; @subtext1: #5c5f77; @subtext0: #6c6f85; @overlay2: #7c7f93; @overlay1: #8c8fa1; @overlay0: #9ca0b0; @surface2: #acb0be; @surface1: #bcc0cc; @surface0: #ccd0da; @base: #eff1f5; @mantle: #e6e9ef; @crust: #dce0e8;
+    @logo-filter: invert(100%) hue-rotate(145deg) invert(100%) sepia(42%) saturate(675%) hue-rotate(178deg) brightness(101%) contrast(92%);
+    @more-filter: invert(100%) invert(100%) sepia(42%) saturate(675%) hue-rotate(178deg) brightness(101%) contrast(92%);
+  };
+  @frappe: {
+    @rosewater: #f2d5cf; @flamingo: #eebebe; @pink: #f4b8e4; @mauve: #ca9ee6; @red: #e78284; @maroon: #ea999c; @peach: #ef9f76; @yellow: #e5c890; @green: #a6d189; @teal: #81c8be; @sky: #99d1db; @sapphire: #85c1dc; @blue: #8caaee; @lavender: #babbf1; @text: #c6d0f5; @subtext1: #b5bfe2; @subtext0: #a5adce; @overlay2: #949cbb; @overlay1: #838ba7; @overlay0: #737994; @surface2: #626880; @surface1: #51576d; @surface0: #414559; @base: #303446; @mantle: #292c3c; @crust: #232634;
+    @logo-filter: invert(100%) saturate(25%) invert(20%) sepia(24%) saturate(560%) hue-rotate(192deg) brightness(90%) contrast(94%);
+    @more-filter: invert(100%) invert(20%) sepia(24%) saturate(560%) hue-rotate(192deg) brightness(90%) contrast(94%);
+  };
+  @macchiato: {
+    @rosewater: #f4dbd6; @flamingo: #f0c6c6; @pink: #f5bde6; @mauve: #c6a0f6; @red: #ed8796; @maroon: #ee99a0; @peach: #f5a97f; @yellow: #eed49f; @green: #a6da95; @teal: #8bd5ca; @sky: #91d7e3; @sapphire: #7dc4e4; @blue: #8aadf4; @lavender: #b7bdf8; @text: #cad3f5; @subtext1: #b8c0e0; @subtext0: #a5adcb; @overlay2: #939ab7; @overlay1: #8087a2; @overlay0: #6e738d; @surface2: #5b6078; @surface1: #494d64; @surface0: #363a4f; @base: #24273a; @mantle: #1e2030; @crust: #181926;
+    @logo-filter: invert(100%) saturate(25%) invert(11%) sepia(47%) saturate(545%) hue-rotate(194deg) brightness(94%) contrast(89%);
+    @more-filter: invert(100%) invert(11%) sepia(47%) saturate(545%) hue-rotate(194deg) brightness(94%) contrast(89%);
+  };
+  @mocha: {
+    @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b;
+    @logo-filter: invert(100%) saturate(0) invert(6%) sepia(50%) saturate(767%) hue-rotate(202deg) brightness(96%) contrast(87%);
+    @more-filter: invert(100%) invert(6%) sepia(50%) saturate(767%) hue-rotate(202deg) brightness(96%) contrast(87%);
```

Comment:
What colors are these actually for? Please name them as `crust-filter` or similar. We will have documentation on this hopefully in the future (see #1089).

Reply:
They're both base, but one throws away the saturation to keep the logo purple and the others keeps the green an red colors without changing for the collapse buttons. Should I name them `@base-logo` and `@base-more`, or something else?

Reply:
What if we used https://github.com/ndmitchell/hoogle/blob/8149c93c40a542bf8f098047e1acbc347fc9f4e6/misc/logo.svg? You could probably do some transformations and get it to be a single path instead of using text elements.

Reply:
Here ya go:
```html
<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 24 24"><path d="M17.71484 8.78125c-.04682 0-.09905.04172-.1582.12305-.17252.23906-.3604.44899-.5625.6289v.2129a1.32 1.32 0 0 1 .12695-.00587c.15034 0 .26489.03307.34375.09961.07887.06655.1795.2501.30274.55078l.24023.57227-.96093 2.55469c-.0764.1947-.14305.31501-.19727.35937-.05422.0419-.13951.07353-.25781.0957v.31055c.13062-.01725.24516-.02539.34375-.02539.10104 0 .25298.00814.45508.0254.19716-.81578.47415-1.76572.83398-2.84766.36969 1.01294.62947 1.68104.77734 2.0039.15034.32286.32432.62962.52149.91797h.11523c.30314-.1528.58734-.2827.85352-.38867v-.20703c-.1528-.01479-.28558-.08236-.39649-.20313-.10844-.12323-.2987-.52485-.57226-1.20507L18.5293 9.8496c-.14788-.36969-.29693-.63854-.44727-.8086-.15034-.17251-.27353-.25976-.36719-.25976M0 8.83789v5.41992h1.09375v-2.3711h2.14453v2.3711h1.09375V8.8379H3.23828v2.13281H1.09375V8.8379zm22.38477 1.71875c-.49785 0-.91915.19603-1.26172.5879-.32533.37707-.48633.81846-.48633 1.32616 0 .56932.16965 1.02657.50976 1.36914.34012.34011.79541.50977 1.3672.50977.56684 0 1.03223-.1917 1.39452-.57617.03944-.0419.0586-.07115.0586-.08594 0-.01232-.04265-.1005-.1289-.26562-.08627-.1676-.13324-.25484-.14063-.25977-.00493-.00493-.0174-.00781-.03711-.00781l-.11914.09765c-.32533.25139-.68287.37696-1.07227.37696-.22674 0-.4376-.07334-.63477-.21875-.20702-.1528-.32733-.3378-.35937-.5547 1.4196-.10597 2.18309-.17736 2.28906-.21679.15773-.06161.23633-.1848.23633-.36719 0-.4732-.15194-.87482-.45508-1.20507-.31053-.34011-.69682-.50977-1.16015-.50977m-6.94336.00781c-.02958 0-.04493.07283-.04493.2207 0 .03451.00288.08869.00782.16016.00493.06901.0078.11835.0078.1504 0 .04435-.00957.0664-.02929.0664-.01725 0-.0465-.02442-.08593-.07617-.23907-.313-.5621-.4707-.96875-.4707-.46088 0-.8347.17542-1.12305.52539-.2711.32778-.40625.72509-.40625 1.19335 0 .46088.14091.85056.42187 1.16602.29329.32779.6671.49023 1.12305.49023.42144 0 .7694-.16533 1.04297-.49804.00986.17991.01367.2959.01367.34765 0 .23414-.0925.41193-.27734.53516-.1602.10844-.36148.1621-.60547.1621-.37215 0-.7436-.10444-1.11328-.3164-.01972-.01232-.03456-.01953-.04688-.01953-.02711 0-.04534.0206-.05273.0625-.07394.39187-.11133.57399-.11133.54688 0 .02464.02781.05533.08203.08984.34504.21195.79796.31836 1.35742.31836.57918 0 .99997-.14761 1.26367-.44336.2514-.2785.37696-.71226.37696-1.29883 0-.29575.00288-.73858.00781-1.33008.00493-.59396.00781-1.04018.00781-1.33593 0-.05176-.01822-.08339-.05273-.0957-.02218-.00987-.153-.03572-.39453-.08008-.24153-.04683-.37575-.07032-.4004-.07032m-8.5918.0293c-.51017 0-.93765.185-1.28516.55469-.33764.36229-.5078.8013-.5078 1.3164s.17016.95216.5078 1.31446c.34504.37461.77253.5625 1.28516.5625.51017 0 .93816-.18933 1.2832-.5664.33765-.36477.50586-.80327.50586-1.31837 0-.51263-.16965-.94876-.50976-1.3086-.34504-.36968-.7716-.55468-1.2793-.55468m3.8711 0c-.51017 0-.93961.185-1.28712.55469-.33764.36229-.50586.8013-.50586 1.3164s.16822.95216.50586 1.31446c.34504.37461.77448.5625 1.28711.5625.51017 0 .93817-.18933 1.2832-.5664.33765-.36477.50587-.80327.50587-1.31837 0-.51263-.16966-.94876-.50977-1.3086-.34504-.36968-.7716-.55468-1.2793-.55468m11.61132.67188c.2021 0 .38473.07859.54492.23632.16266.15774.24414.34037.24414.54492 0 .02958-.0719.0545-.21484.07422-.5077.07394-.94384.10938-1.3086.10938-.08872 0-.1328-.02442-.1328-.07617 0-.22674.08673-.43286.26171-.61524.17745-.18238.37873-.27343.60547-.27343m-7.80664.05273c.26864 0 .48527.10496.6504.31445.15033.19224.2246.42514.2246.70117 0 .26864-.07715.50062-.23242.69532-.1676.21195-.38133.31836-.64258.31836-.2637 0-.4794-.10641-.64453-.31836-.1528-.1947-.22852-.42668-.22852-.69532 0-.27356.07623-.50502.22657-.69726.16512-.21195.3803-.31836.64648-.31836m-7.67578.04101c.29575 0 .54018.11167.73242.33594.18484.21196.27735.46886.27735.76953 0 .29575-.0925.5507-.27735.76758-.1947.2292-.43914.34375-.73242.34375-.29082 0-.5333-.11454-.73047-.34375-.1873-.21688-.28125-.47183-.28125-.76758 0-.30067.0925-.55902.27734-.77343.1947-.22181.43863-.33203.73438-.33203m3.8711 0c.29574 0 .54018.11167.73241.33594.18485.21196.27735.46886.27735.76953 0 .29575-.0925.5507-.27735.76758-.1947.2292-.43913.34375-.73242.34375-.29082 0-.53525-.11454-.73242-.34375-.1873-.21688-.28125-.47183-.28125-.76758 0-.30067.0925-.55902.27735-.77343.1947-.22181.44057-.33203.73632-.33203" fill="purple"/></svg>
```

Reply:
Thanks! This is how it looks with SVGs.

![a screenshot showing the Hoogle logo and the collapse buttons under them](https://github.com/user-attachments/assets/9268cd0b-92c4-4241-b50b-72d482e53930)